android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -25,10 +26,33 @@ class JsonConverter {
          */
         @JvmStatic
         fun toPlayerConfig(json: ReadableMap?): PlayerConfig {
+            var config = PlayerConfig()
             if (json != null && json.hasKey("licenseKey")) {
-                return PlayerConfig(key = json.getString("licenseKey"))
+                config = PlayerConfig(key = json.getString("licenseKey"))
             }
-            return PlayerConfig()
+            if (json != null && json.hasKey("playbackConfig")) {
+                var playbackConfigJson = json.getMap("playbackConfig")
+                if (playbackConfigJson?.hasKey("isAutoplayEnabled") == true) {
+                    config.playbackConfig.isAutoplayEnabled = playbackConfigJson.getBoolean("isAutoplayEnabled")
+                }
+                /**
+                 * Specifies if the player should start muted.
+                 *
+                 * Default is `false`.
+                 */
```

Comment:
I think @rolandkakonyi suggestion referred to all the code comment blocks in the function(s) 🙂